internal/pkg/deploy/cloudformation/stack/stack_integration_test.go

```diff
@@ -1,4 +1,4 @@
-// +build integration,local
+// +build integration localintegration
 // Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Just a thought: would it be helpful if we change the filename such that it's more obvious that *only* local integ tests without AWS calls should be added to this file? So that we won't need to wait until GitHub workflow CI to fail in order to realize that we've put tests in wrong places.

Reply:
yes, that's a good idea!